force-app/main/default/aura/ContactWIList/ContactWIListController.js

```diff
@@ -0,0 +1,23 @@
+({
+	doInit : function(component, event, helper) {
+        var rId = component.get("v.recordId"); //'0033i000001l4NpAAI';
+        console.log(rId);
```

Comment:
Don't forget to remove all console.logs from the code before send it to the repository

Reply:
on it!